ephemeralkey/client.go

```diff
@@ -0,0 +1,53 @@
+// Package ephemeralkey provides the /ephemeral_keys APIs
+package ephemeralkey
+
+import (
+	"fmt"
+
+	stripe "github.com/stripe/stripe-go"
+)
+
+// Client is used to invoke /ephemeral_keys APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// New POSTs new ephemeral keys.
+// For more details see https://stripe.com/docs/api#create_ephemeral_key.
+func New(params *stripe.EphemeralKeyParams) (*stripe.EphemeralKey, error) {
+	return getC().New(params)
+}
+
+func (c Client) New(params *stripe.EphemeralKeyParams) (*stripe.EphemeralKey, error) {
+	if params.StripeVersion == "" {
+		return nil, fmt.Errorf("params.StripeVersion must be specified")
+	}
+
+	body := &stripe.RequestValues{}
+    body.Add("customer", params.Customer)
```

Comment:
So admittedly our convention here is far from perfect, but would you mind changing this to be nil checked?

``` go
	if len(params.Customer) > 0 {
		body.Add("customer", params.Customer)
	}
```